deps.edn

```diff
@@ -160,6 +160,19 @@
  :paths
  ["src" "shared/src" "resources"]
 
+ ;; These are needed for the Athena and Redshift drivers if you are developing against them locally. If those
+ ;; dependencies are not included they are effectively ignored.
+ ;;
+ ;; 1. Maven repos from subprojects do not get copied over -- see
+ ;; https://ask.clojure.org/index.php/10726/deps-manifest-dependencies-respect-repos-dependent-project
+ ;;
+ ;; 2. You cannot include `:mvn/repos` inside of an alias -- see
+ ;; https://ask.clojure.org/index.php/12367/support-mvn-repos-inside-an-alias -- if we could, this could go in the
+ ;; `:drivers` alias instead.
+ :mvn/repos
+ {"athena"   {:url "https://s3.amazonaws.com/maven-athena"}
```

Comment:
This is our s3 bucket, Dan said the name is a little confusing and I agreed. I think he was going to change it to something clearer. But that can be done in a follow-on PR if he doesn't get to it here

Reply:
I don't have the aws setup yet. I think its better to change it now before this ever goes live and anyone looks for artifacts in this bucket.
The idea is that the repo is ours, and might contain any number of artifacts. Naming it after one artifact in it doesn't make sense. Also, the artifact should not have metabase in its name. That is just a hosted jar and we don't want a classpath to end up with `metabase/athena-jdbc` and `amazon/athena-jdbc` at the same time but with the same (or worse, _similar_ but different) class files.

Reply:
I agree with everything you're saying, but my main focus is on getting CI green right now. This PR is supposed to be a collaborative effort as opposed to a Cam solo project, so if you want to see that change then ping ops and get set up on AWS and change it? I have other stuff I need to try to squeeze in for SerDes before 45 so I really shouldn't be spending any more time polishing this